frontend/webpack.config.ts

```diff
@@ -17,6 +17,9 @@ const extractCSS = new MiniCssExtractPlugin({filename: 'app-bundle.css'});
 
 const config: webpack.Configuration = {
   entry: ['./polyfills.js', '@console/app', 'monaco-editor-core/esm/vs/editor/editor.worker.js'],
+  externals: [
+    /prettier/
```

Comment:
The [`externals` config option](https://webpack.js.org/configuration/externals/#externals) is used for imports that are expected to be global (like `$` or `angular`) and will let you `import` them into source code, which is not what we want. So `webpack.IgnorePlugin(/prettier/)` makes more sense here.